imports/startup/server/fixtures.js

```diff
@@ -150,22 +151,23 @@ if (Posts.find().count() === 0) {
         primaryColor: '#fcfcfa',
         secondaryColor: '#807d78'
       }
-    }
+    },
+    isChart: false
   });
 
   Posts.insert({
-    _id: 'WmAF8ovhjhh2cwcrm', 
-    submitted: Date('2017-02-11T02:47:19Z'), 
-    title: 'How the Dog Got Its Bark', 
-    body: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.', 
-    commentCount: 0, 
-    slug: 'hello-world', 
-    userId: 'v7fFs2pJnDnRfmmS7', 
-    author: 'davey', 
-    friendlySlugs: { 
-      slug: { 
-        base: 'hello-world', 
-        index: 0 } 
+    _id: 'WmAF8ovhjhh2cwcrm',
+    submitted: Date('2017-02-11T02:47:19Z'),
+    title: 'How the Dog Got Its Bark',
+    body: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+    commentCount: 0,
+    slug: 'hello-world',
+    userId: 'v7fFs2pJnDnRfmmS7',
+    author: 'davey',
+    friendlySlugs: {
+      slug: {
+        base: 'hello-world',
+        index: 0 }
```

Comment:
How did all those spaces at the line endings get in there? Thanks for trimming those out for me 👍